### PR TITLE
Proof of concept maven.scala

### PR DESCRIPTION
**Pull request by jendap**
This is an answer to @srowen question: "what on earth is pom.scala?"

It shows [polyglot maven](https://github.com/takari/polyglot-maven) is able with a few touches convert and run [apache/spark](https://github.com/apache/spark)

Compare [pom.xml](https://github.com/jendap/spark/blob/master/pom.xml) with [pom.scala](https://github.com/jendap/spark/blob/proof-of-concept_maven.scala/pom.scala) and let me know if you may like it or not.

BEWARE: This is NOT a pull request to spark! This it not even alpha quality. It is not well converted. Not cleaned up. Untouched! There are number of bugs!!!

What works (with maven 3.3.3+):
`mvn clean package -DskipTests`
and
`mvn clean package -DskipTests -Dfoo=2.11.7 -Pscala-2.11 -Dscala.version=2.11.7 -Dscala.binary.version=2.11`

Notes:
- it is pretty nice scala and that is 99% untouched output from automatic translation
- there is a number of small bugs in polyglot maven
- a few things would be nice to discuss and document on upstream polyglot maven before is it adviced for adoption

Future:
- I hope to find time to fix the small bugs and prototype a few things I believe could be better on the current polyglot maven
- nothing upstream for spark or any other project any time soon

Bottom line:
- with two man weeks effort polyglot maven will be ready for something as big as spark

Good job @jvanzyl and @huntc! Thanks!

### CODE REVIEW

**Comment by jendap** [comment]
@srowen check this out! This is not upstream pull request. Just an idea.

Take a (very) brief look - less than a minute - on [pom.xml](https://github.com/jendap/spark/blob/master/pom.xml) vs [pom.scala](https://github.com/jendap/spark/blob/proof-of-concept_maven.scala/pom.scala) and let me know what you think.

"meh" or "bad"? :)
